commands/vpc_peerings.go

```diff
@@ -0,0 +1,271 @@
+	if err != nil {
+		return err
+	}
+	r.Name = name
+
+	vpcIDs, err := c.Doit.GetString(c.NS, doctl.ArgVPCPeeringVPCIDs)
+	if err != nil {
+		return err
+	}
+
+	for _, v := range strings.Split(vpcIDs, ",") {
+		if v == "" {
+			return errors.New("VPC ID is empty")
+		}
+
+		r.VPCIDs = append(r.VPCIDs, strings.TrimSpace(v))
+	}
+
+	if len(r.VPCIDs) != 2 {
+		return errors.New("VPC IDs length should be 2")
+	}
+
+	vpcService := c.VPCs()
+	peering, err := vpcService.CreateVPCPeering(r)
+	if err != nil {
+		return err
+	}
+
+	wait, err := c.Doit.GetBool(c.NS, doctl.ArgCommandWait)
+	if err != nil {
+		return err
+	}
+
+	if wait {
+		notice("VPC Peering creation is in progress, waiting for VPC Peering to become active")
+
+		err := waitForVPCPeering(vpcService, peering.ID, "ACTIVE", false)
+		if err != nil {
+			return fmt.Errorf("VPC Peering couldn't enter `active` state: %v", err)
+		}
+
+		peering, _ = vpcService.GetPeering(peering.ID)
+	}
+
+	item := &displayers.VPCPeering{VPCPeerings: do.VPCPeerings{*peering}}
+	return c.Display(item)
+}
+
+// RunVPCPeeringList lists VPC Peerings
+func RunVPCPeeringList(c *CmdConfig) error {
+	vpcID, err := c.Doit.GetString(c.NS, doctl.ArgVPCPeeringVPCID)
+	if err != nil {
+		return err
+	}
+
+	var list do.VPCPeerings
+	if vpcID == "" {
+		list, err = c.VPCs().ListVPCPeerings()
+		if err != nil {
+			return err
+		}
+	} else {
+		list, err = c.VPCs().ListVPCPeeringsByVPCID(vpcID)
+		if err != nil {
+			return err
+		}
+	}
+
+	item := &displayers.VPCPeering{VPCPeerings: list}
+	return c.Display(item)
+}
+
+// RunVPCPeeringUpdate updates an existing VPC  Peering with new configuration.
+func RunVPCPeeringUpdate(c *CmdConfig) error {
+	err := ensureOneArg(c)
+	if err != nil {
+		return err
+	}
+	peeringID := c.Args[0]
+
+	r := new(godo.VPCPeeringUpdateRequest)
+	name, err := c.Doit.GetString(c.NS, doctl.ArgVPCPeeringName)
+	if err != nil {
+		return err
+	}
+	r.Name = name
+
+	peering, err := c.VPCs().UpdateVPCPeering(peeringID, r)
+	if err != nil {
+		return err
+	}
+
+	item := &displayers.VPCPeering{VPCPeerings: do.VPCPeerings{*peering}}
+	return c.Display(item)
+}
+
+// RunVPCPeeringDelete deletes a VPC Peering by its identifier.
+func RunVPCPeeringDelete(c *CmdConfig) error {
+	if len(c.Args) == 0 {
+		return doctl.NewMissingArgsErr(c.NS)
+	}
+
+	peeringID := c.Args[0]
+
+	force, err := c.Doit.GetBool(c.NS, doctl.ArgForce)
+	if err != nil {
+		return err
+	}
+
+	if force || AskForConfirmDelete("VPC Peering", 1) == nil {
+		vpcs := c.VPCs()
+		if err := vpcs.DeleteVPCPeering(peeringID); err != nil {
+			return err
+		}
+
+		wait, err := c.Doit.GetBool(c.NS, doctl.ArgCommandWait)
+		if err != nil {
+			return err
+		}
+
+		if wait {
+			notice("VPC Peering deletion is in progress, waiting for VPC Peering to be deleted")
+
+			err := waitForVPCPeering(vpcs, peeringID, "DELETED", true)
+			if err != nil {
+				return fmt.Errorf("VPC Peering couldn't be deleted : %v", err)
+			}
+		}
+
```

Comment:
would be great to show a message saying "VPC Peering is successfully deleted" <- something like this